littlepay/api/groups.py

```diff
@@ -11,6 +11,16 @@ class GroupResponse:
     label: str
     participant_id: str
 
+    def csv(self) -> str:
+        """Get a CSV str representation of values for this GroupResponse."""
+        return ",".join(vars(self).values())
```

Comment:
If one of the values contained `,`, would the CSV be incorrect?

(For a group, I think it'd just be `label` that has the potential to contain `,`)

Reply:
Fixed in [b8187d2](https://github.com/cal-itp/littlepay/pull/19/commits/b8187d229627178b882b898f39eaf635eca15488)